lib/index.js

```diff
@@ -24,7 +24,7 @@ class User {
   **/
 
   constructor(opts) {
-    opts.token ? this.token = opts.token : outputFatalError(error.noToken);
+    opts.token ? this.token = opts.token : outputError(error.noToken);
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.

<!-- Comment by [SideCI](https://sideci.com) -->